src/main/resources/META-INF/maven/site.vm

```diff
@@ -137,6 +137,11 @@
             </ul>
 #*      *##end
 ##
+#*      *##if (!$sideBarEnabled)
+          </nav>
+          <div class="nav-collapse">
```

Comment:
I am confused about this. Why is this ecessary now and why not as `nav`?

Reply:
As below...